infra/envs/test/main.tf

```diff
@@ -10,36 +10,69 @@ terraform {
   backend "s3" {
```

Comment:
Do we still need a `test` environment? When we moved accounts, we decided to drop `test` and [only use dev](https://github.com/chanzuckerberg/dcp-prototype/pull/204#issue-389558030). Let me know what you think!

Reply:
I'll continue to use the test environment for the changes I make before make a PR to master. I can removed it from github and use it locally.

Reply:
Sure, keep the test environment then